Antidote/CallsManager.m

```diff
@@ -222,6 +222,23 @@ - (void)updateCurrentCallInterface:(OCTCall *)call
         else {
             activeVC.callDuration = call.callDuration;
         }
+
+        if (call.friendSendingVideo && ! activeVC.videoView) {
+            AALogVerbose(@"provide video feed to video view");
+            activeVC.videoView = [self.manager videoFeed];
+        }
+        else if (! call.friendSendingVideo && ! call.videoIsEnabled) {
```

Comment:
And if you'll change first if there will be no need for `! call.friendSendingVideo` check here.

Also, why do we rely on two different properties `friendSendingVideo` and `videoIsEnabled`. Should there be just one of them?